### PR TITLE
Make guide for tailwind more verbose

### PR DESCRIPTION
**Pull request by duerrsimon**
## Description

The guide for tailwind left out a few details for example that tailwindcss 4.0-alpha has to be used. This makes the instructions a bit more verbose

### CODE REVIEW

File: guides/05_custom-components/05_frontend.md [hunk 1]
```diff
@@ -270,6 +270,15 @@ Svelte options:
 
 The `gradio.config.js` file should be placed in the root of your component's `frontend` directory. A default config file is created for you when you create a new component. But you can also create your own config file and use it to customize your component's build process.
 
+### Example for TailwindCSS
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:273]: Maybe we could name this section 'Using vite plugins' and use tailwind as the example, just so it is clear other vite plugins work.

File: guides/05_custom-components/05_frontend.md [hunk 2]
```diff
@@ -270,6 +270,15 @@ Svelte options:
...
+If you want to use [TailwindCSS](https://tailwindcss.com) you can install it using version 4.0-alpha. 
+
+```
+npm install tailwindcss@next @tailwindcss/vite@next mdsvex
+```
+
+In `gradio.config.js`
 ```typescript
 import tailwindcss from "@tailwindcss/vite";
 import { mdsvex } from "mdsvex";
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:284]: We can remove this.

Comment (pngwn) [reply guides/05_custom-components/05_frontend.md:284]: We can also remove the whole svelte section for this example.

File: guides/05_custom-components/05_frontend.md [hunk 3]
```diff
@@ -283,6 +292,21 @@ export default {
     }
 };
 ```
+Then create a `style.css` file with the following content:
+
+```
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:297]:
```suggestion
```css
```

File: guides/05_custom-components/05_frontend.md [hunk 4]
```diff
@@ -283,6 +292,21 @@ export default {
...
+@import "tailwindcss";
+```
+
+Import this file into `Index.svelte`.
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:301]: Let's make it clear here that you need to import the css file inside the script tag not the style (easy mistake to make).

File: guides/05_custom-components/05_frontend.md [hunk 5]
```diff
@@ -283,6 +292,21 @@ export default {
...
 ```
+Then create a `style.css` file with the following content:
+
+```
+@import "tailwindcss";
+```
+
+Import this file into `Index.svelte`. 
+
+```
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:303]:
```suggestion
```svelte
```

File: guides/05_custom-components/05_frontend.md [hunk 6]
```diff
@@ -270,6 +270,15 @@ Svelte options:
 
 The `gradio.config.js` file should be placed in the root of your component's `frontend` directory. A default config file is created for you when you create a new component. But you can also create your own config file and use it to customize your component's build process.
 
+### Example for TailwindCSS
+
+If you want to use [TailwindCSS](https://tailwindcss.com) you can install it using version 4.0-alpha.
```
Comment (pngwn) [inline guides/05_custom-components/05_frontend.md:275]: I think we should add a paragraph here just explaining that we can use vite plugins that allow you to customise the build process and maybe a link to the vite plugins docs.

**Review by pngwn — COMMENTED** [review]
Thanks for expanding this @duerrsimon! I also think it would be good to add a section on svelte options too (maybe with the mdsvex option) but I am happy to take care of that if it is easier!

File: guides/05_custom-components/05_frontend.md (hunk 2 above)
Comment (duerrsimon) [reply guides/05_custom-components/05_frontend.md]: I modified it a little bit.

File: guides/05_custom-components/05_frontend.md (hunk 6 above)
Comment (duerrsimon) [reply guides/05_custom-components/05_frontend.md]: Done

**Review by aliabd — APPROVED** [review]
Thanks @duerrsimon and thanks @pngwn for making the syntax highlighting changes. Everything lgtm here!

**Pull request merged**